as_impl/src/main/java/ru/mano/aviasales/App.java

```diff
@@ -2,12 +2,33 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import ru.mano.aviasales.entity.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 @SpringBootApplication
 public class App
 {
     public static void main(String[] args)
     {
         SpringApplication.run(App.class, args);
+        // kind of Data Base
+        List<User> users = Arrays.asList(new User(0, User.Role.ADMIN), new User(1), new User(2));
+        List<City> cities = Arrays.asList(new City("Moscow", 0, 0), new City("Samara", 0, 300), new City("Khabarovsk", 0, 1000));
+        List<Ticket> tickets = Arrays.asList(new Ticket(cities.get(0), cities.get(1), 4000), new Ticket(cities.get(1), cities.get(2), 25000), new Ticket(cities.get(2), cities.get(0), 27000));
+
+
+
+        Route route = Route.makeRoute()

```

Comment:
I think... better naming for this method - buildRoute or Route.builder().build();